client/packages/inventory/src/Stocktake/api/operations.graphql

```diff
@@ -159,21 +179,12 @@ mutation upsertStocktakeLines(
         response {
           __typename
           ... on InsertStocktakeLineError {
-            __typename
```

Comment:
This was change to have `__typename` in the type that has unions. It's quite important, since graphql generator adds all variants with __typename to the type, but if __typename is missing, or the type is not matched, it's possible for graphql to return `{}` but f/e would expect `{__typename: SomeType}`, causing runtime errors, I can try and show an example